index.bs

```diff
@@ -575,79 +575,225 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
-		<h4 id="h-navigator-clipboard"><dfn>clipboard</dfn></h4>
-		The [=clipboard=] attribute must return the {{Navigator}}'s [=clipboard=] object.
+		<h4 id="h-navigator-clipboard">clipboard</h4>
+		It is used to execute read/write operations from/to the [=system clipboard=].
+		The {{Navigator/clipboard}} getter steps are to return [=this=]'s [=clipboard=] object.
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
+			typedef Promise&lt;ClipboardItemDataType&gt; ClipboardItemData;
 
-		[SecureContext, Exposed=Window] interface Clipboard : EventTarget {
-		  Promise&lt;ClipboardItems> read();
-		  Promise&lt;DOMString> readText();
-		  Promise&lt;undefined> write(ClipboardItems data);
-		  Promise&lt;undefined> writeText(DOMString data);
-		};
+			[SecureContext, Exposed=Window]
+			interface ClipboardItem {
+				constructor(record&lt;DOMString, ClipboardItemData> items,
+				            optional ClipboardItemOptions options = {});
+
+				readonly attribute PresentationStyle presentationStyle;
+				readonly attribute FrozenArray&lt;DOMString> types;
+
+				Promise&lt;Blob&gt; getType(DOMString type);
+			};
 
-		typedef (DOMString or Blob) ClipboardItemDataType;
-		typedef Promise&lt;ClipboardItemDataType> ClipboardItemData;
+			enum PresentationStyle { "unspecified", "inline", "attachment" };
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+			dictionary ClipboardItemOptions {
+				PresentationStyle presentationStyle = "unspecified";
+			};
+		</pre>
 
-		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
-		    optional ClipboardItemOptions options = {});
-		  static ClipboardItem createDelayed(
-		      record&lt;DOMString, ClipboardItemDelayedCallback> items,
-		      optional ClipboardItemOptions options = {});
+		<dl class=note>
+			<dt><code><var>clipboardItem</var> = new ClipboardItem([<var>items</var>, <var>options</var>])</code>
+			<dd>
+			Creates a new {{ClipboardItem}} object. <var>items</var> are used to fill its MIME types and [=Promise=]s to [=Blob=]s or {{DOMString}}s corresponding to the MIME types, <var>options</var> can be used to fill its {{ClipboardItemOptions}},
+			as per the example below.
 
-		  readonly attribute PresentationStyle presentationStyle;
-		  readonly attribute long long lastModified;
-		  readonly attribute boolean delayed;
+				<pre class="example javascript" highlight=js>
+					const format1 = 'text/plain';
+					const promise_text_blob = Promise.resolve(new Blob(['hello'], {type: format1}));
+					const clipboardItemInput = new ClipboardItem(
+						{[format1]: promise_text_blob},
+						{presentationStyle: "unspecified"});
+				</pre>
```

Comment:
This looks like a note for web-developers. It seems it should be encapsulated in a "domintro" class. See https://wiki.whatwg.org/wiki/Specs/howto#IDL. That will create a "For web developers (non-normative)" box, as it can for instance be seen at https://fetch.spec.whatwg.org/.

`types` and `getType` below should be changed in the same way.

Reply:
For some reason `domintro` doesn't work. Is there anything else that I need to do besides `<dl class=domintro>`?

Reply:
> For some reason `domintro` doesn't work. Is there anything else that I need to do besides `<dl class=domintro>`?

Did you add `<dt>` and `<dd>` elements (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)? See also the code of of `domintro` box used at https://fetch.spec.whatwg.org/#headers-class for an example.

Reply:
It might be a style sheet issue. I'm not sure if all style sheets include the domintro styles.

Reply:
> It might be a style sheet issue. I'm not sure if all style sheets include the domintro styles.

That might indeed be the problem. The `domintro` style used in the fetch standard is defined in https://resources.whatwg.org/standard.css. The Clipboard API spec includes https://www.w3.org/StyleSheets/TR/2021/W3C-ED which imports https://www.w3.org/StyleSheets/TR/2021/W3C-ED/base.css. I didn't know that. I assumed both specs would use the same style sheets, nowadays.
Could it just be a remnant of ancient times? Is it correct that nowadays this w3c spec should use the exact same styles like the whatwg specs?

Reply:
To move _this_ issue forward, perhaps a "note" instead of the "domintro" box suffices for now.

Reply:
Ack.